apache/yunikorn-site
# [YUNIKORN-1502]Chinese translation of Run NVIDIA GPU Jobs in workload

### What is this PR for?
The Chinese translation for the document Run NVIDIA GPU Jobs in workload
in  current user_guide/workload/run_nvidia

### What type of PR is it?

### What is the Jira issue?
[YUNIKORN-1502]Chinese translation of Run NVIDIA GPU Jobs in workload
* https://issues.apache.org/jira/browse/YUNIKORN-1502

### How should this be tested?

./local_build run (make sure all related json files are created in .docusaurus)
./local_build run zh-cn (This step will create additional translated json files in .docusaurus)

### Screenshots 
![image](https://github.com/apache/yunikorn-site/assets/101171023/c0dbf5da-66ba-4d64-bbc8-cc167b216c0f)
